.github/workflows/ci.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           profile: minimal
       - name: install_cargo_deny
         run: |
-          cargo install cargo deny
+          cargo install cargo-deny
       - name: run_cargo_deny_check
```

Comment:
This installs cargo-deny from source; perhaps it would be wise to instead download a binary, either by pre-building it ourselves or using the cargo deny action: https://github.com/EmbarkStudios/cargo-deny-action